2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_setup.py:_flush():79] Configure stats pid to 182885
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_setup.py:_flush():79] Loading settings from /home/joaquin_l_calvo/.config/wandb/settings
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_setup.py:_flush():79] Loading settings from /home/joaquin_l_calvo/Trento/Data_Mining/Data-Mining-Project/src/wandb/settings
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-12-03 15:01:04,626 WARNING MainThread:182885 [wandb_setup.py:_flush():79] Could not find program at -m scripts.run_train_als
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': None, 'program': '-m scripts.run_train_als'}
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_setup.py:_flush():79] Applying login settings: {'api_key': '***REDACTED***'}
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_init.py:_log_setup():533] Logging user logs to /home/joaquin_l_calvo/Trento/Data_Mining/Data-Mining-Project/src/wandb/run-20241203_150104-lspxdi4m/logs/debug.log
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_init.py:_log_setup():534] Logging internal logs to /home/joaquin_l_calvo/Trento/Data_Mining/Data-Mining-Project/src/wandb/run-20241203_150104-lspxdi4m/logs/debug-internal.log
2024-12-03 15:01:04,626 INFO    MainThread:182885 [wandb_init.py:init():619] calling init triggers
2024-12-03 15:01:04,627 INFO    MainThread:182885 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {}
2024-12-03 15:01:04,627 INFO    MainThread:182885 [wandb_init.py:init():669] starting backend
2024-12-03 15:01:04,627 INFO    MainThread:182885 [wandb_init.py:init():673] sending inform_init request
2024-12-03 15:01:04,631 INFO    MainThread:182885 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-12-03 15:01:04,632 INFO    MainThread:182885 [wandb_init.py:init():686] backend started and connected
2024-12-03 15:01:04,641 INFO    MainThread:182885 [wandb_init.py:init():781] updated telemetry
2024-12-03 15:01:04,646 INFO    MainThread:182885 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2024-12-03 15:01:06,002 INFO    MainThread:182885 [wandb_init.py:init():867] starting run threads in backend
2024-12-03 15:01:06,188 INFO    MainThread:182885 [wandb_run.py:_console_start():2456] atexit reg
2024-12-03 15:01:06,188 INFO    MainThread:182885 [wandb_run.py:_redirect():2305] redirect: wrap_raw
2024-12-03 15:01:06,189 INFO    MainThread:182885 [wandb_run.py:_redirect():2370] Wrapping output streams.
2024-12-03 15:01:06,189 INFO    MainThread:182885 [wandb_run.py:_redirect():2395] Redirects installed.
2024-12-03 15:01:06,191 INFO    MainThread:182885 [wandb_init.py:init():911] run started, returning control to user process
2024-12-03 15:01:06,227 WARNING MsgRouterThr:182885 [router.py:message_loop():75] message_loop has been closed
